app/code/Magento/Theme/Block/Html/Header/Logo.php

```diff
@@ -98,7 +98,7 @@ public function getLogoWidth()
                 \Magento\Store\Model\ScopeInterface::SCOPE_STORE
             );
         }
-        return (int)$this->_data['logo_width'] ? : (int)$this->getLogoImgWidth();
```

Comment:
In this case, the value of `design/header/logo_width` config is ignored even when it's set, isn't it?
This is incorrect behavior.